recipe/meta.yaml

```diff
@@ -10,10 +10,10 @@ package:
 
 source:
   url: https://github.com/openmm/openmm-torch/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 06a0f567c1b82746841424d63627e311d93cba4ab0e121f7d9d64701e753467a
+  sha256: f11a2a1f58aa4a58d6c9355f7248a36043966771cb1be028a6f847c258ff4786
```

Comment:
Is there any reason we're using `url` and `sha256` here instead of `git_url` and `git_rev`?  The latter is simpler, since you can just give it a tag name.

Reply:
Because a *conda-forge* person asked so: https://github.com/conda-forge/openmm-torch-feedstock/pull/17.

Reply:
The sha256 here is not the git hash, but the hash of the resulting tarball, to make sure the contents are valid.

Reply:
@peastman can we proceed with the release?